### ISSUE TITLE
radiation.py missing import statement for solar module

### ISSUE THREAD

**Issue by ghost** [pingswept/pysolar#18]
Package Versions: Python 2.7.5+, Pysolar 0.6

I was working through your examples for retrieving the radiation with solar.radiation.GetRadiationDirect(), but the call failed with "NameError: global name 'solar' is not defined".  This is due to referencing day = solar.GetDayOfYear(utc_datetime) in GetRadiationDirect while not having imported the module (doing so would create a circular import between radiation.py and solar.py.)

I'd be happy to code up a fix and submit a pull request if you like.  Otherwise if you need further information please ask.  This is a great utility and I'm considering using it in my work with atmospheric correction.  Thanks!

David

**Comment by karshish** [comment]
Hello,

Sorry if this is not the correct place for this, but it seemed appropriate. I ran into what seems to be the same issue as David reported. I'm calling the function solar.radiation.GetRadiationDirect() and get the error  "NameError: global name 'solar' is not defined".

I'm using Python 2.7.9 and Pysolar-0.6. I tried installing Pysolar-0.6 both by pip and by installing the .zip file downloaded from https://github.com/pingswept/pysolar/releases/tag/0.6 but both installations ran into the same NameError. I import the modules as 'from Pysolar.solar import * '

I was wondering if David's issue was resolved and made its way into Pysolar-0.6?
Thanks in advance - and thanks for creating such a nice module!
